install/install.md

```diff
@@ -10,23 +10,91 @@ description: |
 # RAPIDS Installation Guide
 {: .fs-8 }
 
-RAPIDS has several methods for installation, depending on the preferred environment and versioning. Returning users can jump straight to our **[release selector](#selector)** and new users can get started with these four steps:
+RAPIDS has several methods for installation, depending on the preferred environment and versioning. **New Users** should be mindful of the system and environment prerequisites. 
 {: .fs-6 .fw-300 }
 
-**[1. Provision System](#provision)** <br/>
-**[2A. Set Up Environment](#set-up)** <br/>
-**[2B. Set Up WSL2 Environment](#WSL2)** <br/>
-**[3A. Install RAPIDS (Release Selector)](#selector)** <br/>
-**[3B. Install RAPIDS (pip)](#pip-install)** <br/>
-**[4. Getting Started](#start)**  
+**[Install RAPIDS with Release Selector](#selector)** <br/>
+- [Installation Troubleshooting](#troubleshooting)
+
+**[System Requirements](#system-req)** <br/>
+- [OS, NVIDIA GPU Driver, and CUDA Versions](#system-req)
+- [(Optional) System Recommendations](#system-recommendations)
+- [(Optional) Cloud Instance GPUs](#cloud-gpu)
```

Comment:
Very minor: imo we should remove the (Optional) here - I don't think it adds much. Suggestion added for easy acceptance, but also it could be easy to change my mind here.

```suggestion
- [System Recommendations](#system-recommendations)
- [Cloud Instance GPUs](#cloud-gpu)
```

Reply:
Yes, please drop the (Optional)!